### PR TITLE
Mobile header hero

### PR DESCRIPTION
**Pull request by Drod86**
<h2>In this branch I:</h2>

- Added HTML markup for header and hero sections
- Styled header and hero sections
- Added image files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,17 +1,82 @@
...
+          </li>
+          <li class="socials__icon">
+            <a href="" class="socials__medium">
+              <svg class="icon">
+                <use xlink:href="./images/sprite.svg#medium"></use>
+              </svg>
+            </a>
+          </li>
+        </ul>
+      </main>
```
Comment (usorfaitheloho) [inline index.html]: Kindly use a `section` tag for this [part of your project as the container of a section is a `section` tag, you can check this link for more details [url](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Drod86  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Nice UI design ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Drod86) [reply index.html]: @usorfaitheloho  thank you for your suggested change. I changed the hero section tag from a main tag to a section tag.

**Review by Strangeal — APPROVED** [review]
Hi @Drod86, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**